src/Utility/browser.ts

```diff
@@ -89,3 +91,17 @@ export const clientPost = (
     form.submit();
     document.body.removeChild(form);
 };
+
+// This uses the `navigator.language` property (similar-ish to the `Accept-Language` header which we cannot access from
+// JS) which may not necessarily represent the user's country (or even include region information at all).
+// The more reliable way would be to feed the user's IP into a geolocation service but that is not an option, so we have
+// to stick with this.
+export const guessUserCountry = (): Country => {
+    const { country } = parseBcp47Tag(navigator.language) as { country: Country };
+
+    // If we cannot guess the user's country, it makes sense to fallback to the language.
+    if (!navigator.language || !country) return fallback_countries[window.LOCALE];
```

Comment:
Can we assume `window.LOCALE` to always be well defined? Should we harden our code for that? Depending on that, I would expect this to be undefined in some cases.

Reply:
Yes, on second thought I realized that this falls out of hugo…